.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
+__pycache__
+env
```

Comment:
@ashryaagr, @siddhantkhandelwal: Add the migrations folder to this list.

Reply:
Sure, will be taken care of. Thanks!

Reply:
Sounds good, thanks!